pkg/helpers/match.go

```diff
@@ -47,3 +47,8 @@ func MapHasKey(m *map[string]string, label string) bool {
 	}
 	return ans
 }
+
+func IsDomainName(hostname string) bool {
```

Comment:
unit tests missing

Reply:
also, might worth checking if we can re-use https://github.com/asaskevich/govalidator#isurl instead, so we don't need to maintain this piece of code :+1:

Reply:
I tested with the isurl method and seams to work, even i am not 100% confident to be honest. I could not really track the implementation of what it actually checks, but the basic tests I did it worked out. I also had a look to `isDNSName` method but it does not force having a top level domain (at least one `.`, so it does not work for us).

So for now I am using this `govalidator.isurl`

Reply:
looks good actually! :+1: